.circleci/config.yml

```
@@ -493,7 +493,7 @@ jobs:
           fi
           echo "Parallel backend flags: "${PARALLEL_FLAGS}
 
-          export COMMAND='((echo "export BUILD_ENVIRONMENT=${BUILD_ENVIRONMENT}" && echo '"$PARALLEL_FLAGS"' && echo "source ./workspace/env" && echo "sudo chown -R jenkins workspace && cd workspace && .jenkins/pytorch/build.sh") | docker exec -u jenkins -i "$id" bash) 2>&1'
+          export COMMAND='((echo "export BUILD_ENVIRONMENT=${BUILD_ENVIRONMENT}" && echo '"$PARALLEL_FLAGS"' && echo "source ./workspace/env" && echo "sudo chown -R jenkins workspace && cd workspace && .jenkins/pytorch/build.sh && find /var/lib/jenkins/workspace -type f -name "*.a" -exec rm {} \; && find /var/lib/jenkins/workspace -type f -name "*.o" -exec rm {} \;") | docker exec -u jenkins -i "$id" bash) 2>&1'
```

3 nits:
 - Using `|xargs rm` is much faster than `-exec rm {} \;` as it spawns a new process only once instead of 10K times
 - Limit the `find` to  `workspace/build` folder
 - Use `-or` to merge to find operators together
```suggestion
          export COMMAND='((echo "export BUILD_ENVIRONMENT=${BUILD_ENVIRONMENT}" && echo '"$PARALLEL_FLAGS"' && echo "source ./workspace/env" && echo "sudo chown -R jenkins workspace && cd workspace && .jenkins/pytorch/build.sh && find /var/lib/jenkins/workspace/build -type f -name "*.a" -or -name "*.o"|xargs rm") | docker exec -u jenkins -i "$id" bash) 2>&1'
```

---

1. I see your "|xargs rm" and I raise you a "-delete". I found an article that claims it's more efficient with -delete
2. If the image is just to run a Python wheel, why not remove all the .o and .a? They won't be needed anyway. The space gain with the reduced scope is about 100 MB compared to 300 MB with the previous scope.
3. Done. Didn't know about -or. Thanks!